src/opm/parser/eclipse/EclipseState/Schedule/TimeMap.cpp

```diff
@@ -121,7 +123,16 @@ namespace {
             if (keyword.name() == "DATES") {
                 for (size_t recordIndex = 0; recordIndex < keyword.size(); recordIndex++) {
                     const auto &record = keyword.getRecord(recordIndex);
-                    const std::time_t nextTime = TimeMap::timeFromEclipse(record);
+                    std::time_t nextTime;
+
+                    try {
+                        nextTime = TimeMap::timeFromEclipse(record);
+                    } catch (const std::exception& e) {
+                        const OpmInputError opm_error { e, keyword.location() } ;
+                        OpmLog::error(opm_error.what());
+                        std::throw_with_nested(opm_error);
```

Comment:
You strictly speaking need to `#include <exception>` to use `std::throw_with_nested()`.

Reply:
Fixed